Source/Header/DaySymbolsView.swift

```diff
@@ -3,7 +3,15 @@ import UIKit
 
 public final class DaySymbolsView: UIView {
   public private(set) var daysInWeek = 7
-  private var calendar = Calendar.autoupdatingCurrent
+  public var calendar = Calendar.autoupdatingCurrent {
+    didSet {
+      // TODO: I honestly don't know if I should be doing this. Changes to
```

Comment:
It's correct approach. API allows changing of the `Calendar`, therefore, anything could change, including the index of the first weekday.